django_fido/views.py

```diff
@@ -180,12 +183,13 @@ def complete_registration(self, form: Form) -> AttestationObject:
     def form_valid(self, form: Form) -> HttpResponse:
         """Complete the registration and return response."""
         try:
-            attestation = self.complete_registration(form)
+            auth_data = self.complete_registration(form)
```

Comment:
Return type is marked as `AttestationObject` but it doesn't seem to be that case.

Reply:
Fixed.